File: csharp/QWORDSCSHARP/Services/WordSelectionService.cs
public class WordSelectionService
{
    private WordList wordlist;
    private string selectedWord;

    public WordSelectionService()
    {
        this.wordlist = new WordList();
        this.selectedWord = wordlist.GetRandomWord();
    }

    public string GetWord()
    {
        return this.selectedWord;
    }
}

File: nodejs/services/WordSelectionService.js
const WordList = require('../repositories/WordList');

class WordSelectionService {
  constructor() {
    this.wordlist = new WordList();
    this.selectedWord = this.wordlist.getRandomWord();
  }

  getWord() {
    return this.selectedWord;
  }
}

module.exports = WordSelectionService;
